app/src/main/res/layout/view_allpatientsinfacility.xml

```diff
@@ -0,0 +1,46 @@
+<?xml version="1.0" encoding="utf-8"?>
+<org.simple.clinic.allpatientsinfacility.AllPatientsInFacilityView xmlns:android="http://schemas.android.com/apk/res/android"
+  xmlns:tools="http://schemas.android.com/tools"
+  android:layout_width="match_parent"
+  android:layout_height="match_parent"
+  android:animateLayoutChanges="true">
+
+  <LinearLayout
+    android:id="@+id/noPatientsContainer"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    android:layout_gravity="center"
+    android:orientation="vertical"
+    android:visibility="gone"
+    tools:visibility="visible">
+
+    <TextView
+      android:id="@+id/noPatientsLabel"
+      android:layout_width="match_parent"
+      android:layout_height="wrap_content"
+      android:gravity="center"
+      android:textAppearance="@style/Clinic.V2.TextAppearance.H6.Grey1"
+      tools:text="No patients at CHC Bhucho"
+      android:layout_marginStart="@dimen/spacing_56"
+      android:layout_marginEnd="@dimen/spacing_56"/>
+
+    <TextView
+      android:layout_width="match_parent"
+      android:layout_height="wrap_content"
+      android:layout_marginTop="@dimen/spacing_12"
+      android:gravity="center"
+      android:text="@string/allpatientsinfacility_nopatients_message"
+      android:textAppearance="@style/Clinic.V2.TextAppearance.Subtitle1Center.Grey1"
+      android:layout_marginStart="@dimen/spacing_32"
+      android:layout_marginEnd="@dimen/spacing_32"/>
+
+  </LinearLayout>
+
+  <androidx.recyclerview.widget.RecyclerView
+    android:id="@+id/patientsList"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:visibility="gone"
```

Comment:
Instead of setting visibility here, please ensure that the controller sets the initial state correctly. We should avoid setting visibility in the XML because that is state contained in XML now.

Reply:
Makes sense, I'll do this.